src/main/java/org/tdl/vireo/service/SubmissionEmailService.java

```diff
@@ -0,0 +1,288 @@
+package org.tdl.vireo.service;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Service;
+import org.tdl.vireo.model.EmailRecipient;
+import org.tdl.vireo.model.EmailRecipientAssignee;
+import org.tdl.vireo.model.EmailRecipientContact;
+import org.tdl.vireo.model.EmailRecipientOrganization;
+import org.tdl.vireo.model.EmailRecipientPlainAddress;
+import org.tdl.vireo.model.EmailRecipientSubmitter;
+import org.tdl.vireo.model.EmailRecipientType;
+import org.tdl.vireo.model.EmailTemplate;
+import org.tdl.vireo.model.EmailWorkflowRule;
+import org.tdl.vireo.model.FieldPredicate;
+import org.tdl.vireo.model.InputType;
+import org.tdl.vireo.model.Submission;
+import org.tdl.vireo.model.User;
+import org.tdl.vireo.model.repo.ActionLogRepo;
+import org.tdl.vireo.model.repo.EmailTemplateRepo;
+import org.tdl.vireo.model.repo.FieldPredicateRepo;
+import org.tdl.vireo.model.repo.InputTypeRepo;
+import org.tdl.vireo.utility.TemplateUtility;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import edu.tamu.weaver.email.service.EmailSender;
+
+/**
+ * Provide e-mail sending specific to the Submission process.
+ *
+ * E-mails are built utilizing SimpleMailMessage and then sent via EmailSender.
+ *
+ * @see EmailSender
+ * @see SimpleMailMessage
+ */
+@Service
+public class SubmissionEmailService {
+
+    private Logger LOG = LoggerFactory.getLogger(this.getClass());
+
+    @Autowired
+    private ActionLogRepo actionLogRepo;
+
+    @Autowired
+    private EmailSender emailSender;
+
+    @Autowired
+    private EmailTemplateRepo emailTemplateRepo;
+
+    @Autowired
+    private FieldPredicateRepo fieldPredicateRepo;
+
+    @Autowired
+    private InputTypeRepo inputTypeRepo;
+
+    @Autowired
+    private TemplateUtility templateUtility;
+
+    /**
+     * Manually send the e-mails to the advisors for a given Submission.
+     *
+     * @param user Associated User.
+     * @param submission Associated Submission.
+     */
+    public void sendAdvisorEmails(User user, Submission submission) {
+        InputType contactInputType = inputTypeRepo.findByName("INPUT_CONTACT");
+        EmailTemplate template = emailTemplateRepo.findByNameAndSystemRequired("SYSTEM Advisor Review Request", true);
+
+        String subject = templateUtility.compileString(template.getSubject(), submission);
+        String content = templateUtility.compileTemplate(template, submission);
+
+        List<String> fullRecipientList = new ArrayList<>();
+
+        // TODO: this needs to only send email to the advisor not any field value that is contact type
```

Comment:
I can see that this was carried over from the original implementation, but I think it might be the time to fix it. We don't want the entire committee getting the adviser approval link or anything like that.

@jeremythuff @wwelling @jsavell @jcreel Thoughts on this?

Reply:
This needs a fix soon.  Let's resolve this question and address the problem in the context of issues of #1303 and #1305 .